### PR TITLE
Update README.md with basic Usage information

### PR DESCRIPTION
**Pull request by chrismytton**
Part of https://github.com/everypolitician/everypolitician/issues/299
Fixes https://github.com/everypolitician/wikisnakker/issues/21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,40 @@ And then execute:
 
 ## Usage
 
-TODO: Write usage instructions here
+### Single item
+
+You can get a Wikidata item using its qualifier. For example [Douglas Adams'](https://www.wikidata.org/wiki/Q42) has the qualifier `Q42`.
```
Comment (tmtmtmtm) [inline README.md]: Don't think we need a trailing quote on "Douglas Adams" there…

**Comment by tmtmtmtm** [comment]
I think we probably need to show an example of using the plural form to fetch all the values for something, rather than just the single value case — this should really be the default approach, with the singular as a fallback when you just want the first one.

**Comment by tmtmtmtm** [comment]
@chrismytton — would be good to get something merged here, as I keep going to read the instructions, discovering there's none, and then finally remembering that this Pull Request is here…

**Comment by chrismytton** [comment]
> I think we probably need to show an example of using the plural form to fetch all the values for something, rather than just the single value case — this should really be the default approach, with the singular as a fallback when you just want the first one.

Good point, I've added an example of doing this now.

I've also rearranged things slightly to (hopefully!) make it clearer that looking up multiple items at once is what this library is good at and should be the default mode of operation for most people.

**Pull request merged**